draft-irtf-cfrg-opaque.md

```diff
@@ -368,55 +368,28 @@ We write `(kU, _) = KeyGen()` to denote use of this function for generating secr
 # Core Protocol {#protocol}
 
 OPAQUE consists of two stages: registration and authenticated key exchange.
-In the first stage, a client registers its password with the server and stores its encrypted credentials on the server.
-In the second stage, a client obtains those credentials, unlocks them using the user's password and subsequently uses
+In the first stage, a client registers its password with the server and stores
+its encrypted credentials on the server. In the second stage, a client obtains
+those credentials, unlocks them using the user's password and subsequently uses
 them as input to an authenticated key exchange (AKE) protocol.
 
 Both registration and authenticated key exchange stages require running an OPRF protocol.
 The latter stage additionally requires running a mutually-authenticated
 key-exchange protocol (AKE) using credentials recovered after the OPRF protocol completes.
-(The key-exchange protocol MUST satisfy forward secrecy and the KCI requirement discussed in {{security-considerations}}.)
+(The key-exchange protocol MUST satisfy forward secrecy and the KCI requirement
+discussed in {{security-considerations}}.)
 
 We first define the core OPAQUE protocol based on a generic OPRF, hash, and MHF function.
 {{instantiations}} describes specific instantiations of OPAQUE using various AKE protocols,
 including: HMQV, 3DH, and SIGMA-I. {{I-D.sullivan-tls-opaque}} discusses integration with
 TLS 1.3 {{RFC8446}}.
 
-## Protocol messages {#protocol-messages}
+## Data types {#data-types}
 
-The OPAQUE protocol runs the OPRF protocol in two stages: registration and
-authenticated key exchange. A client and server exchange protocol messages in
-executing these stages. This section specifies the structure of these protocol
-messages using TLS notation (see {{RFC8446}}, Section 3).
```

Comment:
Perhaps we want to keep this sentence:

> This section specifies the structure of these protocol
messages using TLS notation (see {{RFC8446}}, Section 3).

Since it still applies for the remainder of the document